extra_features_cargo_install.sh

```diff
@@ -0,0 +1 @@
+cargo install --path . --force --features=extra
```

Comment:
With recent `cargo`, you shouldn't need the `--force` flag any more. Do you see errors without it?

Reply:
Nope. In fact I always use the `cargo install nu --features=extra`. Just copied and pasted the above mentioned command from the book.

If you wish, I would totally remove it.

Reply:
yeah, let's go ahead and remove it and I'll remove it from the book.